misc/docerina/src/main/java/org/ballerinalang/docgen/Generator.java

```diff
@@ -794,7 +794,7 @@ private static String getDocCodeBlockString(MarkdownCodeBlockNode markdownCodeBl
             doc.append(codeLineNode.codeDescription().toString());
         }
 
-        doc.append(markdownCodeBlockNode.endBacktick().text());
+        doc.append(markdownCodeBlockNode.endBacktick().text()).append("\n");
```

Comment:
Sorry didn't notice this earlier. Shall we use `toString` here? it includes the newline as well.

```suggestion
        doc.append(markdownCodeBlockNode.endBacktick().toString());
```